graph_explorer/src/graph_explorer/graph_visualizer/templates/workspace_config.html

```diff
@@ -41,16 +41,17 @@ <h1>Workspace Configuration</h1>
         </select>
       </div>
       {% for param in parameters %} {% if param.1 == "str" %}
-      <input name="{{ param.0 }}" class="input" type="text" placeholder="{{ param.0 }}" />
+      <input name="{{ param.0 }}" class="input" type="text" placeholder="{{ param.0 }}" value="{{ param.2 }}" />
```

Comment:
Curious about why this doesn't throw an exception because when we are creating a new workspace "param.2" is not defined.

Reply:
If a variable does not exist, `django` will insert an empty string by default.